shared/src/main/java/net/skinsrestorer/shared/storage/Config.java

```diff
@@ -42,7 +42,7 @@
     public static List<String> CUSTOM_GUI_SKINS = null;
     public static boolean PER_SKIN_PERMISSIONS = false;
     public static int SKIN_EXPIRES_AFTER = 20;
-    public static boolean MULTI_BUNGEE_ENABLED = false;
+    public static boolean FORWARD_TEXTURES = false;
```

Comment:
must be true, auto updaters will not have the config.

Reply:
Oh oops my bad.

Reply:
> Did forget it because on default spigot it just takes the default value from the default config, not from a class 😅 We should do that in the future as well. Kinda don't like that we have the same default values declared twice.

There might be a difference on when we want to have new users by default test a new functionality ant not current users so we don't create issues on a large scale.